### PR TITLE
PR: feat/files into Dev

### PR DESCRIPTION
**Pull request by lpendem**

### CODE REVIEW

File: controllers/permits/applications/special-uses/utility.js [hunk 1]
```diff
@@ -19,11 +19,12 @@ const include = require('include')(__dirname);
 const path = require('path');
 const AWS = require('aws-sdk');
 const errors = require('./patternErrorMessages.json');
+const async = require('async');
 
 //*************************************************************
 // AWS
 
-const AWS_ACCESS_KEY_ID = process.env.AWS_ACCESS_KEY_ID;
+const AWS_ACCESS_KEY_ID = process.env.AWS_ACCESS_KEY_ID1;
```
Comment (cameronwolf) [inline controllers/permits/applications/special-uses/utility.js]: Shouldn't this be `AWS_ACCESS_KEY_ID`?

File: controllers/permits/applications/special-uses/commercial/temp-outfitters/index.js [hunk 2]
```diff
@@ -119,103 +118,166 @@ function fileErrors(missingFiles){
 	output = output.trim();
 	return output;
 }
+
+function concatErrors(errorMessages){
```
Comment (cameronwolf) [inline controllers/permits/applications/special-uses/commercial/temp-outfitters/index.js]: Also, just looking at code coverage, this function is not being tested at all. Creating a function in util that buildErrorMessage also uses will help up the function coverage.

Comment (lpendem) [inline controllers/permits/applications/special-uses/commercial/temp-outfitters/index.js]: @cameronwolf concatErrors moved to utility, and buildErrorMessage hasn't updates though

File: controllers/permits/applications/special-uses/utility.js (hunk 1 above)
Comment (lpendem) [inline controllers/permits/applications/special-uses/utility.js]: @cameronwolf this was for my localhost fix, updated.

**Pull request merged**